packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2560,25 +2591,75 @@ describe("Runtime", () => {
 					},
 					provideEntryPoint: mockProvideEntryPoint,
 				});
+				runtimes.set(containerRuntime.clientId, containerRuntime);
 				logger.clear();
```

Comment:
Doesn't seem good to allow two of these to have undefined clientId. Will they be undefined ever? If not capture and assert not undefined.